skycultures/arabic/constellation_names.eng.fab

```diff
@@ -0,0 +1,138 @@
+0100 "الفرقدان" _("The Two Oryx Calves")
+0200 "فأس الرحى" _("The Axe Of The Grindstone")
+# TRANSLATORS: Na'sh is a proper name meaning bier
+0300 "بنات نعش" _("Daughters Of Na'sh")
+0400 "الظباء وأولاد الظباء" _("The Gazelles And Thier Youngsters")
+0410 "القفزة الأولى" _("The First Leap")
+0420 "القفزة الثانية" _("The Second Leap")
+0430 "القفزة الثالثة" _("The Third Leap")
+0501 "الفارطان" _("The Two Preceding To The Water")
+0500 "الحوض" _("The Pool")
+0600 "الذئبان" _("The Two Wolves")
+0601 "أظفار الذئب" _("The Wolf Claws")
+0700 "العوائذ" _("The Mother Camels")
+0800 "الأثافي" _("The Three Stone Support")
+0900 "القدر" _("The Pot")
+1000 "الراعي الشمالي والشاء" _("The Shepherd And The Sheep")
+1001 "كوكبا الفرق" _("The Two Separated Stars")
+1100 "ذو السلاح" _("The Armed One")
+1101 "الرمح" _("The Spear")
+1200 "السماك الأعزل" _("The Unarmed One")
+1300 "الضباع وأولاد الضباع" _("The Hyenas And Their Youngsters")
+#3 page
+1400 "قصعة المساكين" _("The Bowl Of The Poor")
+1500 "الروضة والأغنام" _("The Desert Garden And The Goats")
+1501 "النسق الشآمي" _("The Northern Line")
+1502 "النسق اليماني" _("اthe Southern Line")

```

Comment:
strange first character in translatable string. Maybe "The ..."

Reply:
Sorry, remnant of the original native language file.
Fixed.
I added references. Beta75 solves issue #2055 .